model/schema/core.yaml

```diff
@@ -476,24 +476,27 @@ classes:
       In the context of this model, this entity is used as a landmark for the
       location of other genomic entities.
     slots:
-      - display_name
+      - name
       - genome_assembly
+      - maps_to_chromosome
     slot_usage:
-      curie:
+      name:
```

Comment:
just making sure you wanted to remove the curie field requirement here?

Reply:
yes, curie is reserved for AGR curies and not currently a required field